app/views/partials/branchProtection.scala.html

```diff
@@ -0,0 +1,85 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.connector.GitRepository
+@import uk.gov.hmrc.cataloguefrontend.EnableBranchProtection
+@import views.html.helper.CSRF
+@import uk.gov.hmrc.cataloguefrontend.routes
+
+@(repo: GitRepository, authorisation: EnableBranchProtection.HasAuthorisation, request: Request[_])
+
+@boolToGlyph(bool: Boolean) = {
+  @if(bool) {
+    <span class="glyphicon glyphicon-ok text-success"></span>
+    <span class="sr-only">enabled</span>
+  } else {
+    <span class="glyphicon glyphicon-remove text-danger"></span>
+    <span class="sr-only">not enabled</span>
+  }
+}
+
+<h5 style="margin-left: 19.2px">Branch Protection</h5>
+<ul class="list list--minimal">
+    <li class="list-item">@boolToGlyph(repo.branchProtection.exists(_.requiresApprovingReviews)) Requires approving reviews</li>
+    <li class="list-item">@boolToGlyph(repo.branchProtection.exists(_.dismissesStaleReviews)) Dismisses stale reviews</li>
+    <li class="list-item">@boolToGlyph(repo.branchProtection.exists(_.requiresCommitSignatures)) Requires commit signatures</li>
+</ul>
+
+@authorisation match {
+  case EnableBranchProtection.HasAuthorisation.Yes if repo.branchProtection.forall(!_.isProtected) => {
+    <button type="button"
+            class="btn btn-primary"
+            data-toggle="modal"
+            data-target="#enableBranchProtectionModal"
+            style="margin-left: 15px; margin-bottom: 15px">
+        Enable
+    </button>
+  }
+  case _ => {
+    <button type="button"
+            class="btn btn-primary"
+            data-toggle="modal"
+            data-target="#enableBranchProtectionModal"
+            style="margin-left: 15px; margin-bottom: 15px"
+            disabled>
```

Comment:
can we just wrap the filter around the `disabled` property (less code, and easier to see the difference).
with a boolean wrapper it might be simpler 
```scala
@(!authorisation.asBoolean || repo.branchProtection.forall(!_.isProtected)) { disabled }
```
Is it worth not having the button at all if you don't have perms - might make it clearer to the user when they don't have the perms vs already enabled